packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -419,35 +419,50 @@ export class AtomicSearchBox {
     this.ariaMessage = '';
   }
 
+  private makeSuggestionPart(isSelected: boolean, isDivider: boolean) {
+    let part = 'suggestion';
+    if (isSelected) {
+      part += ' active-suggestion';
+    }
+    if (isDivider) {
+      part += ' suggestion-divider';
+    }
+    return part;
+  }
+
   private renderSuggestion(
-    suggestion: SearchBoxSuggestionElement,
-    index: number
+    item: SearchBoxSuggestionItem,
+    index: number,
+    lastIndex: number
   ) {
     const id = `${this.id}-suggestion-${index}`;
     const isSelected = id === this.activeDescendant;
+    const isLast = index === lastIndex;
+    const isDivider = isDividerElement(item);
```

Comment:
The divider could be hidden whenever there are no associated items, rather than "if is last"